analyses/malawi/dryspells_trigger/exploration/compare_arc2_chirps.Rmd

```diff
@@ -0,0 +1,483 @@
+spell **after** it has been detected by ARC2. This is reassuring news for the switch
+to ARC2 for the observational trigger. However, what does it look like where
+CHIRPS has data and ARC2 is missing it?
+
+### Geographic comparison
+
+```{r geo-compare}
+arc2_missing_plot <- arc2_chirps_df %>%
+  filter(chirps_ds, !arc2_touch_ds) %>%
+  group_by(pcode) %>%
+  summarize(count = n()) %>%
+  right_join(adm2, by = c("pcode" = "ADM2_PCODE")) %>%
+  st_as_sf() %>%
+  ggplot() +
+  geom_sf(aes(fill = count)) +
+  scale_fill_gradient(na.value = "white",
+                      breaks = 1:5) +
+  geom_sf(data = adm1, fill = NA, size = 0.75) +
+  coord_sf(datum = NA) +
+  theme_minimal() +
+  labs(title = "Dry spells only detected using CHIRPS") +
+  theme(plot.title.position = "plot")
+
+chirps_missing_plot <- arc2_chirps_df %>%
+  filter(!chirps_ds, arc2_touch_ds, arc2_centroid_ds) %>%
+  group_by(pcode) %>%
+  summarize(count = n()) %>%
+  right_join(adm2, by = c("pcode" = "ADM2_PCODE")) %>%
+  st_as_sf() %>%
+  ggplot() +
+  geom_sf(aes(fill = count)) +
+  scale_fill_gradient(na.value = "white",
+                      breaks = 1:5) +
+  geom_sf(data = adm1, fill = NA, size = 0.75) +
+  coord_sf(datum = NA) +
+  theme_minimal() +
+  labs(title = "Dry spells only detected using ARC2",
+       subtitle = "(both centroid and touching method)") +
+  theme(plot.title.position = "plot")
+
+plot_grid(arc2_missing_plot, chirps_missing_plot)
+```
+
+```{r geo-compare-arc2}
+centroid_missing_plot <- arc2_chirps_df %>%
+  filter(arc2_centroid_ds, !arc2_touch_ds) %>%
+  group_by(pcode) %>%
+  summarize(count = n()) %>%
+  right_join(adm2, by = c("pcode" = "ADM2_PCODE")) %>%
+  st_as_sf() %>%
+  ggplot() +
+  geom_sf(aes(fill = count)) +
+  scale_fill_gradient(na.value = "white",
+                      breaks = 1:5) +
+  geom_sf(data = adm1, fill = NA, size = 0.75) +
+  coord_sf(datum = NA) +
+  theme_minimal() +
+  labs(title = "Dry spells only detected using centroids") +
+  theme(plot.title.position = "plot")
+
+touch_missing_plot <- arc2_chirps_df %>%
+  filter(!arc2_centroid_ds, arc2_touch_ds) %>%
+  group_by(pcode) %>%
+  summarize(count = n()) %>%
+  right_join(adm2, by = c("pcode" = "ADM2_PCODE")) %>%
+  st_as_sf() %>%
+  ggplot() +
+  geom_sf(aes(fill = count)) +
+  scale_fill_gradient(na.value = "white",
+                      breaks = 1:5) +
+  geom_sf(data = adm1, fill = NA, size = 0.75) +
+  coord_sf(datum = NA) +
+  theme_minimal() +
+  labs(title = "Dry spells only detected using touching") +
+  theme(plot.title.position = "plot")
+
+plot_grid(centroid_missing_plot, touch_missing_plot)
+```
+
+### Trigger activation
+
+Let's consider if we're using these datasets for trigger activation and how they would
+compare. We will only consider dry spells that are confirmed at some point in January
+and February. Let's just quickly look at the number of dry spells detected this time period
+for each method to start.
+
+```{r trigger-chirps}
+chirps_df %>%
```

Comment:
These are still throughout all of MWI, correct? ie. not filtered to just the southern region

Reply:
Good point. Might compare just across the southern region further on.

Reply:
Added in update and even without explicitly filtering by region, the only confirmed triggers in those months were in the southern region.